build.sh

```diff
@@ -259,46 +260,49 @@ function copyResources {
 	fi
 }
 
+function makeToolbarIcons {
+	browser="$1"
+	icon_dir="$BUILD_DIR/$browser/images/toolbar"
+	mkdir -p "$icon_dir"
+	
+	# Check for rsvg-convert
+	if ! command -v rsvg-convert --version &> /dev/null ; then
+		echo ""
+		echo "rsvg-convert is not available"
+		usage
+	fi
+	
+	set -e
+	for f in $COLLECTION_IMAGES $TOOLBAR_IMAGES
+	do
+		rsvg-convert $f -w 32 -h 32 -o "$icon_dir/"`basename $f @2x.svg`".png"
```

Comment:
Note that `COLLECTION_IMAGES` are *not* @2x. I'm not sure if it's because of that, but in the build folder the collection SVGs are converted to `collection.svg.png` so they don't show up